README.md

```diff
@@ -8,13 +8,11 @@
 
```

Comment:
Could you run `npx kcd-scripts format` to fix all the formatting? I think you editor messed up a few things...

Reply:
done :)

Reply:
Hmmm.... This is weird... It's formatting things completely different for lists... Did prettier change how it formats lists?